docs/1.5/antora.yml

```
@@ -8,7 +8,7 @@ nav:
 - modules/en/nav.adoc
 asciidoc:
   attributes:
-    elemental_slack_name: #elemental
+    elemental_slack_name: Elemental
```

```suggestion
    elemental_slack_name: #elemental
```

I don't think needs to be changed based on the how it's used. 

E.g. on `pages/index.adoc` the snippet where it's used says `...reach out to the {elemental_slack_url}[{elemental_slack_name}] Slack channel.`

With the change, it meaning changes to `The Slack channel for Elemental` whereas the original seems like it's intended to mean `go to this specific channel on our Slack` (although it may be more clear if it was `#elemental channel on the Rancher Users Slack` or something).

---

Thanks for reviewing Billy! I adjusted the attribute phrasing to your suggestion in the antora.yml file per version and updated the index.adoc in this commit for reference:https://github.com/rancher/elemental-product-docs/pull/37/commits/d05db25441fc5a07cac3168e90467bccead9cf67.